coffee_testing/src/lib.rs

```diff
@@ -100,6 +101,19 @@ impl CoffeeTesting {
         })
     }
 
+    // init coffee in a tmp directory with arguments.
+    pub async fn tmp_with_args(args: &CoffeeTestingArgs) -> anyhow::Result<Self> {
+        let temp_dir_path = args.data_dir.clone();
+        let dir = Builder::new().prefix(&temp_dir_path).tempdir()?;
```

Comment:
this is always the correct path?
```suggestion
        let dir = Builder::new().prefix(&temp_dir_path).tempdir()?;
```

Reply:
we can take the temp path as an argument to avoid this line of code

Reply:
why not put the tmp path directly inside the args?

Reply:
The args expects a string for the data directory.
what do you think of the latest commit?
I added an info debug statement to make sure they are the same

Reply:
>The args expects a string for the data directory.

but you can convert the tmp directory to a path and then convert the path to a string no? as I did here https://github.com/coffee-tools/coffee/blob/master/coffee_testing/src/lib.rs#L90